.travis.yml

```diff
@@ -13,11 +13,3 @@ script:
   - make -C examples/linear_algebra test
   - make -C examples/resample benchmark
   - doxygen Doxyfile
-
-deploy:
```

Comment:
Why remove this? I think it is working well and we should keep it.

Reply:
Oops, I meant to submit a PR from another branch, not master... I disabled that part for my own travis tests (not sure how to test otherwise).

Submitted a revert on this PR to get moving.